plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -969,6 +972,10 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             ((==) @Data)
             (runCostingFunTwoArguments . paramEqualsData)
+    toBuiltinMeaning SerialiseData =
+        makeBuiltinMeaning
+            (BS.toStrict . serialise @Data)
```

Comment:
Indeed it is not worth it, because the most likely thing to do after calling `serialiseData` is to hash the result, e.g. `sha3_256(serialiseData(I(3)))`. All our hash builtins (coming from cardano-crypt-class) go through strict bytestrings, since as @kwxm pointed out, they are foreign-imported from C.

Reply:
Yeah, this seems fine.